examples/rmg/superminimal/sensitivity_analysis_superminimal_RMS.ipynb

```diff
@@ -25,74 +25,112 @@
-    "`mechanism_path`: path to the generated rms input file\n",
+    "## Load the mechanism \n",
+    "<p style=\"color: red;\"> [User input required] </p>\n",
     "\n",
-    "`initial_conds`: a dictionary containing the initial conditions, where\n",
-    "- `T`: temperature (K)\n",
-    "- `P`: pressure (Pa)\n",
-    "- `H2`: number of moles of H2\n",
-    "- `O2`: number of moles of O2\n",
+    "Load the mechanism file from a `.rms` file\n",
+    "```\n",
+    "phase_dict = rms.readinput(\"/rmg/RMG-Py/examples/rmg/superminimal/rms/chem13.rms\")\n",
+    "```\n",
+    "where `mechanism_path` is the path to the generated rms input file\n",
+    "\n",
+    "P.S. The same function also works for chemkin files (with thermochemistry and kinetics in the same file):\n",
     "\n",
-    "`tspan`: the simulation time span ($t_0$, $t_f$) in second"
+    "```\n",
+    "phase_dict = rms.readinput(\"/rmg/RMG-Py/examples/rmg/superminimal/chemkin/chem_annotated.inp\"; \n",
+    "                           spcdict=\"chemkin/species_dictionary.txt\")\n",
+    "```"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 2,
-   "id": "4aed6687",
+   "id": "76557080",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "RDKit WARNING: [21:48:22] WARNING: not removing hydrogen atom without neighbors\n",
+      "[21:48:22] WARNING: not removing hydrogen atom without neighbors\n"
+     ]
+    }
+   ],
    "source": [
-    "mechanism_path = \"rms/chem13.rms\"\n",
-    "initial_conds = {\"T\": 1000.0, \"P\": 10.0e5, \"H2\": 0.67, \"O2\": 0.33}\n",
-    "tspan = (0.0, 0.02)"
+    "mechanism_path = \"/rmg/RMG-Py/examples/rmg/superminimal/rms/chem13.rms\"  # You may change the path to actual mechanism file path\n",
+    "\n",
+    "phase_dict = rms.readinput(mechanism_path)\n",
+    "spcs = phase_dict[\"phase\"][\"Species\"]\n",
+    "rxns = phase_dict[\"phase\"][\"Reactions\"]"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "87400737",
+   "id": "c1fa0586",
    "metadata": {},
    "source": [
-    "### Load the mechanism"
+    "If you haven't generate the superminimal RMG mechanism yet, remove the `#` in the following cell and run it. This is an example for docker users."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 3,
-   "id": "d2f20f2e",
+   "id": "f9caf172",
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "RDKit WARNING: [00:02:46] WARNING: not removing hydrogen atom without neighbors\n",
-      "[00:02:46] WARNING: not removing hydrogen atom without neighbors\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "phase_dict = rms.readinput(mechanism_path)\n",
-    "spcs = phase_dict[\"phase\"][\"Species\"]\n",
-    "rxns = phase_dict[\"phase\"][\"Reactions\"]"
+    "# ! python-jl /rmg/RMG-Py/rmg.py /rmg/RMG-Py/examples/rmg/superminimal/input.py"
```

Comment:
Can you move the python-jl cell before the mechanism loading cells, as you did in the ch402cat notebook?